mkt/site/middleware.py

```diff
@@ -111,7 +111,7 @@ def process_request(self, request):
             if stored_ov_lang.lower() in settings.LANGUAGE_URL_MAP:
                 ov_lang = stored_ov_lang
 
-        if 'lang' in request.REQUEST:
+        if 'lang' in request.GET:
```

Comment:
I looked in fireplace and couldn't fine any uses of passing lang as anything but a GET parameter in the route args.